app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+
+import { Formik, Form } from "formik";
+import { Button, Pane, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import {
+  CONTACTS_FORM_VALIDATION_SCHEMA,
+  ROLES,
+  PROFILE_PICTURE_URL,
+} from "../constants";
+
+const ContactForm = ({ onClose, contact, setContacts }) => {
+  const [submitted, setSubmitted] = useState(false);
+
+  const handleSubmit = values => {
+    setContacts(contacts => [
+      {
+        ...values,
+        id: contacts.length + 1,
+        role: values.role.label,
+        imageUrl: PROFILE_PICTURE_URL,
+        createdAt: new Date(),
+      },
+      ...contacts,
+    ]);
```

Comment:
What is the reason for setting the new contact as the first added contact and not as the last added contact? Meaning here you are always adding the new contact to index 0. Why so? What's the advantage?

Reply:
This was made so for the development easiness. If this was not done then I need to scroll down after creating each contact to verify whether it worked or not.
On the contrary, ideally in the real world, I think the opposite is true.